modules/tinymce/src/core/main/ts/lists/lists/NodeType.ts

```diff
@@ -64,21 +62,21 @@ const isChildOfBody = (dom: DOMUtils, elm: Element): boolean =>
   dom.isChildOf(elm, dom.getRoot());
 
 export {
-  isTextNode,
+  isBlock,
```

Comment:
There is duplication here between this module and the NodeType module. But we could log that as a cleanup ticket.